spot-contracts/contracts/FeePolicy.sol

```diff
@@ -77,13 +79,9 @@ contract FeePolicy is IFeePolicy, OwnableUpgradeable {
     ///      Adds a safety buffer to ensure that rollovers are better sustained.
     uint256 public targetSubscriptionRatio;
 
-    /// @notice The lower bound of deviation ratio, below which some operations (which decrease the dr) are disabled.
-    uint256 public deviationRatioBoundLower;
-
-    /// @notice The upper bound of deviation ratio, above which some operations (which increase the dr) are disabled.
-    uint256 public deviationRatioBoundUpper;
-
-    //-----------------------------------------------------------------------------
+    /// @notice The lower and upper deviation ratio bounds outside which
+    ///         some operations (like flash swaps) which move dr away from 1.0 are disabled.
+    Range public drHardBounds;
```

Comment:
Making 2 uints into a struct with 2 uints should be storage safe. But we should test this on a mainnet fork before deploying.